thirdparty/sdl/joystick/SDL_gamepad.c

```diff
@@ -2676,6 +2676,19 @@ bool SDL_ShouldIgnoreGamepad(Uint16 vendor_id, Uint16 product_id, Uint16 version
         // Don't treat the Wii extension controls as a separate gamepad
         return true;
     }
+    if (SDL_strstr("TouchPad", name) || // Matches e.g. "SynPS/2 Synaptics TouchPad"
+        SDL_strstr("Touchpad", name) || // Matches e.g. "Sony Interactive Entertainment DualSense Wireless Controller Touchpad"
+        SDL_strstr("Synaptics", name) || // Matches e.g. "Synaptics TM2768-001". Depending on Linux kernel version, Synaptics touchpads don't always have "touchpad" in their name.
+        SDL_strstr("Trackpad", name) ||
+        SDL_strstr("Clickpad", name) ||
+        SDL_strstr("Keyboard", name) || // Matches e.g. "PG-90215 Keyboard", "Usb Keyboard Usb Keyboard Consumer Control", "Framework Laptop 16 Numpad Module System Control", "Framework Laptop 16 Keyboard Module - ISO System Control"
+        SDL_strstr("Mouse", name) || // Matches e.g. "Mouse passthrough"
+        SDL_strstr("Pen", name) || // Matches e.g. "Wacom One by Wacom S Pen"
+        SDL_strstr("Finger", name) || // Matches e.g. "Wacom HID 495F Finger"
+        SDL_strstr("Led", name)) { // Matches e.g. "ASRock LED Controller"
```

Comment:
```suggestion
        SDL_strstr(name, "LED")) { // Matches e.g. "ASRock LED Controller"
```
Should it be all uppercase because "LED" is uppercase in the device's name?

Reply:
Yes, the check is case-sensitive. Fixed.